### PR TITLE
Add a menu position example

### PR DESCRIPTION
**Pull request by ljhaywar**
This is an example of how to set the menu position of the Grammarly button. We don't currently have one in the docs, and I need one for an upcoming blog post.

You can preview this at 
https://codesandbox.io/s/github/grammarly/grammarly-for-developers/tree/menu-position-example/examples/editor-sdk-menu-position?file=/public/index.html

### CODE REVIEW

**Comment by AndrewDiMola** [comment]
Awesome! We should add to https://github.com/grammarly/grammarly-for-developers/tree/main/examples as well.

**Comment by AndrewDiMola** [comment]
I understand why we're placing the button in the center, but I think we should also explicitly call out that we're doing this because the setting to the position to the right, while the button is in the bottom-right, will still open the menu to the left. Also because all other examples have it in a predictable position and it may look like this is not intentional.

image

**Comment by AndrewDiMola** [comment]
We also have some updated guidance on how we refer to and stylize copy like "web component" and `<grammarly-button>`.

CC: @paigedr

File: examples/editor-sdk-menu-position/readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+This demo shows how to set the [menu position](https://developer.grammarly.com/docs/api/editor-sdk/grammarlybuttonelement#menuposition) of the Grammarly button in the [Grammarly Text Editor SDK](https://developer.grammarly.com/).
+
+## Try the demo
+
+You can try the demo in [CodeSandbox](https://codesandbox.io/s/github/grammarly/grammarly-for-developers/tree/main/examples/editor-sdk-menu-position?file=/public/index.html).
+
+## How it works
+
+[index.html](./public/index.html) and the other HTML files in this repository explicitly add the `<grammarly-button>` element to the page. The element includes the `menu-position` attribute, setting it to either `left` or `right`.
```
[the remark this answers is not in the document]
Comment (AndrewDiMola) [reply examples/editor-sdk-menu-position/readme.md:11]: https://github.com/grammarly/grammarly-for-developers/pull/452#issuecomment-1402630524

**Comment by ljhaywar** [comment]
Thanks for that great feedback, @AndrewDiMola! I've made some updates. Can you take another look?

I opened PAR-2095 to track updating the API Reference.

File: examples/editor-sdk-menu-position/public/index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link rel="stylesheet" href="/style.css" />
+    <title>Grammarly Button Menu Position</title>
+  </head>
+  <body>
+    <h1>Grammarly Button Menu Position</h1>
+    <p>
+      When you use the <code>&lt;grammarly-button&gt;</code> web component to
```
[the remark this answers is not in the document]
Comment (ljhaywar) [reply examples/editor-sdk-menu-position/public/index.html:15]: @paigedr Should this be 
When you use the <code>grammarly-button</code> web component...

File: examples/editor-sdk-menu-position/public/index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      place the Grammarly button in a custom position, you can choose whether
+      the button menu appears to the left or right of the button. By default,
+      the menu is placed to the left of the button.
+    </p>
+    <p>
+      Select the Grammarly button beneath the textarea. Notice that the menu
```
[the remark this answers is not in the document]
Comment (ljhaywar) [reply examples/editor-sdk-menu-position/public/index.html:21]: To make sure... You recommend using "text area" in this case?

**Comment by ljhaywar** [comment]
@AndrewDiMola @paigedr I'm going to merge this, so we'll have it ready for tomorrow's stream. Let me know if you see anything that needs to be updated, and I'll fix it tomorrow morning.

File: examples/editor-sdk-menu-position/public/index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (paigedr) [reply examples/editor-sdk-menu-position/public/index.html]: Yes!

**Pull request merged**